imports/anim/client.lua

```diff
@@ -0,0 +1,67 @@
+---@class AnimProperties
+---@field ped number
+---@field dict string
+---@field anim string
+---@field options table
```

Comment:
This needs to be properly defined, not just a generic table.

Reply:
this as well as all your other comments should be resolved in latest commit